components/apim_appgw/.terraform-version

```diff
@@ -0,0 +1 @@
+1.4.6
```

Comment:
You don't need this - the pipeline uses tfswitch and will pull the terraform version from the `required_version` property

Reply:
done